src/main/java/edu/cornell/kfs/pdp/batch/service/impl/CuExtractPaymentServiceImpl.java

```diff
@@ -170,11 +163,10 @@ protected void writeExtractBundledAchFile(PaymentStatus extractedStatus, String
            }
            writeCloseTag(os, 0, "achPayments");
            
-           // send summary email
            getPaymentFileEmailService().sendAchSummaryEmail(unitCounts, unitTotals, SpringContext.getBean(DateTimeService.class).getCurrentDate());
```

Comment:
Looks like this should be referencing dateTimeService (or getDateTimeService() if you prefer) to avoid the SpringContext.getBean call.